claims/opportunity-for-gain.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
I think we are struggling a bit here to really clarify *what* claim is being made as evidenced by "evidence of claim" section. Is this that it is a good investment, that it is an exciting speculative investment, that unregulated markets are a good idea ...

If the latter i think we want to focus on that and the claim is not "opportunity to make money" but "unregulated trading in securities is desirable". I think this latter is probably the focus actually for now (as that was the core of the deep dive actually)

Reply:
@rufuspollock I agree that trying to address the "unregulated crypto markets are desirable" claim in the same claim page as "opportunity for gain" isn't working. As you suggested, I have focussed on the "unregulated trading in securities is desirable" claim and have turned the securities deep dive into the claim page for that claim: https://github.com/life-itself/web3/commit/c7c28503a725a77d8aa0ee64d33c008733242b45 . I will now address the opportunity for gain claim in its own page. Hopefully this will make things clearer and easier.